airbyte-integrations/connectors/source-awin-advertiser/manifest.yaml

```diff
@@ -0,0 +1,308 @@
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/campaign_performance"
+    transactions:
+      type: DeclarativeStream
+      name: transactions
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /advertisers/{{ config['advertiserId'] }}/transactions/
+          http_method: GET
+          request_parameters:
+            "showBasketProducts\t": "True"
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        decoder:
+          type: JsonDecoder
+      incremental_sync:
+        type: DatetimeBasedCursor
+        cursor_field: transactionDate
+        lookback_window: P{{ config['lookback_days'] }}D
+        cursor_datetime_formats:
+          - "%Y-%m-%dT%H:%M:%S"
+        datetime_format: "%Y-%m-%dT%H:%M:%S"
+        start_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ config[\"start_date_transactions\"] }}"
+          datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+        start_time_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: startDate
+        end_time_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: endDate
+        end_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ now_utc().strftime('%Y-%m-%dT%H:%M:%S') }}"
+          datetime_format: "%Y-%m-%dT%H:%M:%S"
+        step: "{{ config['step_increment'] }}"
+        cursor_granularity: P1D
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/transactions"
+  base_requester:
+    type: HttpRequester
+    url_base: https://api.awin.com
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config[\"api_key\"] }}"
+
+streams:
+  - $ref: "#/definitions/streams/campaign_performance"
+  - $ref: "#/definitions/streams/transactions"
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - advertiserId
+      - api_key
+      - step_increment
+      - lookback_days
+      - start_date_campaigns
+      - start_date_transactions
+    properties:
+      advertiserId:
+        type: string
+        name: advertiserId
+        order: 0
+        title: advertiserId
+      api_key:
+        type: string
+        order: 1
+        title: API Key
+        airbyte_secret: true
+      step_increment:
+        type: string
+        description: ISO8601 duration format like 'P400D'
+        title: Step Increment
+        default: P400D
+        order: 2
+      lookback_days:
+        type: integer
+        description: Number of days to re-fetch on each sync.
+        title: Lookback Days
+        order: 3
+      start_date_campaigns:
```

Comment:
@ryanmcg2203 let's use one start date. it follow the standard from other connectors.